tls/s2n_connection.h

```diff
@@ -85,6 +101,11 @@ struct s2n_connection {
     uint8_t actual_protocol_version;
     uint8_t actual_protocol_version_established;
 
+    /* Certificate Authentication and Verification Parameters */
+    s2n_cert_auth_type client_cert_auth_type;
```

Comment:
I think it's reasonable to leave it as is then. Maybe we can revisit it when we start shoring up our client code?